client/src/components/marketLocator/map.tsx

```diff
@@ -1,130 +1,136 @@
-import React, { useState } from "react";
+import TextField from '@material-ui/core/Input';
```

Comment:
For our build time, we need to be consistent with an updated version of Material UI
```suggestion
import TextField from '@mui/material/TextField';
```